### PR TITLE
new: SECURITY disclosure policy

### PR DESCRIPTION
**Pull request by leodido**
Introducing a security policy for disclosure and management of reported vulnerabilities.

This file, when merged, will act as security policy for all the **falcosecurity** organization projects.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Security updates will typically only be applied to the latest release (at least until **Falco** reaches first stable major version).
+
+| Version.   | Supported          |
+| ---------- | ------------------ |
+| >=0.15.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+To report a security issue, email opensource@sysdig.com and include the word "SECURITY" in the subject line.
```
Comment (leodido) [inline SECURITY.md]:
```suggestion
To report a security issue, email [opensource@sysdig.com](mailto:opensource@sysdig.com?subject=SECURITY) and include the word "SECURITY" in the subject line.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+
+## Supported Versions
+
+Security updates will typically only be applied to the latest release (at least until **Falco** reaches first stable major version).
+
+| Version.   | Supported          |
+| ---------- | ------------------ |
+| >=0.15.x   | :white_check_mark: |
```
Comment (fntlnz) [inline SECURITY.md]: I don't understand why 0.15.0 here and not 0.18.0 - we don't backport security fixes to older releases at this stage.

**Review by fntlnz — COMMENTED** [review]
Looks very good overall, just have a question then LGTM

File: SECURITY.md (hunk 2 above)
Comment (leodido) [reply SECURITY.md]: Right! Fixing it

Comment (leodido) [inline SECURITY.md]:
```suggestion
| >=0.18.x   | :white_check_mark: |
```

**Review by fntlnz — APPROVED** [review]

**Pull request merged**